### ISSUE TITLE
Wrong enum list generation on graphql type 

### ISSUE THREAD

**Issue by MeXaaR** [Cauen/prisma-generator-pothos-codegen#12]
Hi, this plugin is awesome and it helps me a lot. Thank you very much for that.

I noticed a problem on the generation wich cause me to not be able to query the data after I created it.

this is my prisma schema 
```
model TestCampaign {
  id   String @id @default(cuid())
  Unit Unit   @relation(fields: [unitId], references: [id])

  unitId       String
  ownerId      Int
  status       TestStatus @default(REQUESTED)
  test_request String
  type         TestType[]
  startedAt    DateTime?
  endedAt      DateTime?

  @@index([unitId])
}

enum TestStatus {
  REQUESTED
  IN_PROGRESS
  FINISHED
}

enum TestType {
  LOAD_OPTIMIZATION
  CONTROL
  PERFORMANCE
  MISC
}
```

the generation problem is on this line
```  type         TestType[]```
I can create the document because it expects an array. but I can't query it because it wants to give an enum key.

in the object.base.ts, this is generated
```
export const TestCampaignTypeFieldObject = defineFieldObject('TestCampaign', {
  type: Inputs.TestType,
  description: undefined,
  nullable: false,
  resolve: (parent) => parent.type,
});
```

there is a ts error on parent.type saying that TestType[] is cot compatible with
"'MaybePromise<"LOAD_OPTIMIZATION" | "CONTROL" | "PERFORMANCE" | "MISC">'"
Which is pretty logic because I asked for an array in my schema.

When I query a TestCampaign, I have this error
Enum "TestType" cannot represent value: ["CONTROL"]
The weird thing is I can create it without a problem

so I check on Apollo Studio and TestCampaign has this schema generated
```
type TestCampaign {
  Unit: Unit!
  endedAt: DateTime
  id: ID!
  ownerId: Int!
  startedAt: DateTime
  status: TestStatus!
  test_request: String!
  type: TestType!     // this should be [TestType]!
  unitId: String!
}
```

and this is my pothos config

```
// ./pothos.config.js
/** @type {import('prisma-generator-pothos-codegen').Config} */
module.exports = {
    inputs: {
        outputFilePath: './src/graphql/generated/inputs.ts',
    },
    crud: {
        outputDir: './src/graphql/generated/',
    },
    global: {
        builderImporter: `import { builder } from '../builder';`,
    },
};
```

Nothing fancy. 

so anyone has an idea whats missing ?

Thanks guys

**Comment by Cauen** [comment]
Hi @MeXaaR 
Thanks for testing and contributing by reporting bugs
Tested your use case and fixed it in the new version 0.4.5

Can you please test if it's working correctly now?

**Comment by MeXaaR** [comment]
awesome!! thanks
